meta-rpi-extras/recipes-bsp/boot-files/rpi-config_%.bbappend

```diff
@@ -0,0 +1,6 @@
+
+#
+# Reserve 512MB RAM for the GPU if this is the QtAS spin, otherwise 128MB
```

Comment:
I think we use the term variant instead of spin. Not sure though.

Reply:
Good point, we speak of variants in the SWF and in the `conf/` directory, so we should be consistent.